Example/Podfile.lock

```diff
@@ -289,17 +289,17 @@ SPEC CHECKSUMS:
   ORStackView: b9507271cb41fb9e0b3eecc6414d831201e7cf7c
   Pulley: 2679a2b5a9714c5eb760b1fbb535d15c60e50fb4
   React: 9d063e2f356c8cd2f54dd550d4507740037cbabe
-  react-native-cameraroll: 4c5ca0eab943949e7c39fa549d8f442dae9d3776
+  react-native-cameraroll: b1faef9f2ea27b07bdf818b8043909a93f797eca
```

Comment:
I'm not sure if this will have unintended effects on other installs-- cc @ashfurrow is it OK for this file to change?

Reply:
Hmm! It _should_ be okay. What I'd want to look out for is churn in this file (lots of PRs that all change these lines). These values are hashes of the podspecs (similar to gemspecs) and so constantly churning values would indicate that someone's local machine is for some reason generating different hashes.

tl;dr this is totally fine to merge, let's just keep an eye out for further churn in this file 👍